### ISSUE TITLE
[Feature Request] Endless scrolling

### ISSUE THREAD

**Issue by z775729168** [hkalexling/Mango#137]
**Is your feature request related to a problem? Please describe.**
It’s annoying to click the next entry every time.

**Describe the solution you'd like**
The first thought is to add a shortcut key, but why not just cancel the next entry button, let the scrolling be infinite, such as loading next chapter on the last page.

**Describe a small use-case for this feature request**
save some time, keep your hands on the keyboard, focus on reading.

**Additional context**
n/a

**Comment by hkalexling** [comment]
Good idea, but I am worried that if the user keeps scrolling, we will have a huge DOM, and we might have some memory issues on low-end devices. Ideally, we should remove the pages that are not in view from the DOM, but that would be quite complicated as most pages are server-rendered, and we are not using a JS framework like Vue/React. Let me think about it and see what we can do!
